### DISCUSSION TITLE
Is it possible to intercept all traffic  ?

### DISCUSSION THREAD

**Discussion by ghost** [gravitational/teleport#12877] [General]
hi
To access the infrastructure of the company (where I work remotely), I need to use the commands

tsh login --proxy teleport.mycompany.site 
tsh app login git

Can my employer intercept my browser traffic?
Is this even possible after setting up such a proxy connection via teleport?

Thanks

**Comment by webvictim** [comment]
It's technically possible for a Teleport cluster administrator to inspect all the traffic sent to and from it, assuming that they control the private key for the TLS certificates that secure the web UI and that they have access to the signing keys for Teleport's CAs in its backend database. This would include any applications served on a Teleport subdomain (like the `git.teleport.mycompany.site` in your example). Teleport does not facilitate or allow inspection of any traffic not destined for a Teleport cluster.

Whether your employer is actually doing this is a different question entirely. It isn't trivial to do.
